pkg/tsdb/loki/testdata/streams_simple.golden.txt

```diff
@@ -4,34 +4,34 @@ Frame[0] {
     "executedQueryString": "Expr: query1"
 }
 Name: {level="error", location="moon"}
-Dimensions: 2 Fields by 1 Rows
```

Comment:
How do you actually generate these files?

Reply:
you just write the test, and run the test. it will complain that the snapshot-file could not be found, but it will also create the snapshot-file. then you commit the snapshot file 😄